src/Network/Network/AzureFirewallPolicy/ApplicationRuleCondition/NewAzureFirewallPolicyApplicationRuleCommand.cs

```diff
@@ -70,6 +70,12 @@ public override void Execute()
         {
             base.Execute();
 
+            // One of SourceAddress or SourceIpGroup must be present
+            if ((SourceAddress == null) && (SourceIpGroup == null))
```

Comment:
Also create two parameter sets for this condition

Reply:
Done